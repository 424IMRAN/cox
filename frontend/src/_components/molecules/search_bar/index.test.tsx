import {render,fireEvent,screen, queryByPlaceholderText} from '@testing-library/react';
import SearchAppBar from './index'

describe('Searchbar test',()=>{
    const handleChange=jest.fn();
    const component=(<SearchAppBar  />);

    it('Should match the snapshot',()=>{
        const {container}=render(component);
        expect(container.firstChild).toMatchSnapshot();
    });

})