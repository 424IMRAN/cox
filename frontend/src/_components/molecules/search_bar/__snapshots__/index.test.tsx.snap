// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbar test Should match the snapshot 1`] = `
<div
  class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
>
  <div
    class="makeStyles-search-1"
  >
    <div
      class="MuiInputBase-root"
    >
      <input
        class="MuiInputBase-input makeStyles-inputInput-4"
        placeholder="Search..."
        type="text"
        value=""
      />
    </div>
    <div
      class="makeStyles-searchIcon-2"
    >
      <img
        alt="icon"
        class="makeStyles-Icon-3"
        data-testid="svgicon"
        height="16.93px"
        src="[object Object]"
        width="16.93px"
      />
    </div>
  </div>
</div>
`;
